Makefile

```diff
@@ -1,14 +1,50 @@
 # SPDX-FileCopyrightText: © 2023 Merqury Cybersecurity Ltd <info@merqury.eu>
 # SPDX-License-Identifier: AGPL-3.0-only
 
-.PHONY: db_container db_start db_stop db_clean_container db_clean_container_and_data build build_release run_tests clean
+# include ./.env
+
+DATABASE_PORT?=10000
+DATABASE_HOST?=127.0.0.1
+DATABASE_USER?=db_user
+DATABASE_PASSWORD?=db_password
+DATABASE_URL?=postgres://${DATABASE_USER}:${DATABASE_PASSWORD}@${DATABASE_HOST}:${DATABASE_PORT}/key_store
+ETSI_014_REF_IMPL_PORT_NUM?=8443
+ETSI_014_REF_IMPL_IP_ADDR?=${DATABASE_HOST}
+ETSI_014_REF_IMPL_NUM_WORKER_THREADS?=2
+CERTS_DIR?=$(abspath ./certs)
```

Comment:
_question_ does this makefile work even if called from a different directory than the one it is in?

Reply:
Correct, this won't work when calling the make from another directory: I will fix this